app/service/RequestService.scala

```diff
@@ -1,16 +1,15 @@
 package service
 
-import java.util.Random
-
 import javax.inject.{Inject, Singleton}
 import play.api.libs.ws.WSClient
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.Future
 
 @Singleton
-class RequestService @Inject()(wsClient: WSClient)(implicit ec: ExecutionContext){
+class RequestService @Inject()(wsClient: WSClient) {
 
   def makeRequest(url: String): Future[String] = {
+      import scala.concurrent.ExecutionContext.Implicits._
```

Comment:
Do you think that default scala's execution context is tunned better?

Reply:
It's more that you don't need an implicit parameter on the method, as it could potentially be called with the wrong implicit execution context i.e. one tuned for IO.

If you're doing a map that is CPU bound, then the default fork/join pool is the right choice.  You can also pass in the injected `ExecutionContext` which goes to the Akka default dispatcher:

```scala
class RequestService @Inject()(wsClient: WSClient)
                           (implicit ec: ExecutionContext) {

} 
```